src/cascadia/TerminalControl/ControlCore.h

```diff
@@ -80,11 +80,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         void SendInput(const winrt::hstring& wstr);
         void PasteText(const winrt::hstring& hstr);
-        bool CopySelectionToClipboard(bool singleLine, const Windows::Foundation::IReference<CopyFormat>& formats);
+        bool CopySelectionToClipboard(bool singleLine, const Windows::Foundation::IReference<CopyFormat>& formats, bool clearSelection = false);

```

Comment:
```suggestion
        bool CopySelectionToClipboard(bool singleLine, const Windows::Foundation::IReference<CopyFormat>& formats, bool clearSelection = true);
```
Yeah, I think I need to change this to `true`? Testing before committing

Reply:
for the record, I have it set to true in ControlInteractivity. So one of these is _definitely_ wrong